### PR TITLE
doc: add minutes for meeting 27 Oct 2021

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: meetings/2021-10-27.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+## Agenda
+
+## Announcements
+
+*Extracted from **next10-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* build: provide pre-built nodejs/node images or cache  [#39672](https://github.com/nodejs/node/issues/39672)
+  * Continuing to work on it. Have a good image which is a good dev image
```
Comment (sheplu) [inline meetings/2021-10-27.md:27]:
```suggestion
  * Continue to work on it. Have a good image which is a good dev image
```

File: meetings/2021-10-27.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      future Node.js success
+
+* deep dive on Maintaining/growing the level of contribution in future Next-10 meeting  [#86](https://github.com/nodejs/next-10/issues/86)
+  * Challenges in contributing
+    * Assumed, harder to get investment both company and individual
+    * Requires sustained contribution over time
+    * Two types of contributions
+      * Scratching and itch/solving personal problem
+      * Moving the platform forward
+  * The Moving the platform forward
```
Comment (sheplu) [inline meetings/2021-10-27.md:56]: duplicate here

**Review by sheplu — CHANGES_REQUESTED** [review]

File: meetings/2021-10-27.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
```
Comment (mhdawson) [inline meetings/2021-10-27.md:56]:
```suggestion
```

**Comment by mhdawson** [comment]
@sheplu updated.

**Review by sheplu — APPROVED** [review]
LGTM

**Pull request merged**